packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -296,12 +299,43 @@ export const DataGrid: React.FC<GridProps> = (props) => {
     return { onChange: handlePageChange, pageSize: values?.pageSize };
   }, [values?.hidePagination, values?.pageSize]);
 
+  const onSortAction = useEnsembleAction(onSort);
+  // page change action
+  const onSortActionCallback = useCallback(
+    (sorter: SorterResult<unknown>) => {
+      if (onSortAction) {
+        onSortAction.callback({
+          sort_order: sorter.order,
+          column_title: sorter.column?.title,
```

Comment:
Can we use camelCase here for consistency? I think we need to pass in the data key for the column being sorted, not just the title.

Reply:
```js
{
    "sorter": {
        "column": {
            "key": "0",
            "dataIndex": "dummyData",
            "title": "id"
        },
        "order": "ascend",
        "field": "dummyData",
        "columnKey": "0"
    }
}
```

@evshi We are getting this information from sorter, so based on this we can pass data. Now here what you mean by data key property ?

Reply:
Each column is bound to a key in the data object. For example if my data object was { foo: 0, bar: "baz" }, then foo and bar would be the keys.